site/src/components/Welcome/Welcome.tsx

```diff
@@ -31,22 +30,23 @@ export const Welcome: FC<
           }}
         />
       </div>
-      <Typography
-        css={css`
-          text-align: center;
-          font-size: 32px;
-          font-weight: 400;
-          margin: 16px 0 32px;
-          line-height: 1.25;
+      <h1
+        css={{
+          textAlign: "center",
+          fontSize: 32,
+          fontWeight: 400,
+          margin: 0,
+          marginTop: 16,
+          marginBottom: 32,
+          lineHeight: 1.25,
 
-          & strong {
-            font-weight: 600;
-          }
-        `}
-        variant="h1"
+          "& strong": {
+            fontWeight: 600,
+          },
+        }}
```

Comment:
aww, why get rid of the `css`? I kinda like those for when we have nested selectors